### PR TITLE
Added timestamp and file extension to the file name when output is file

### PR DESCRIPTION
**Pull request by juanarmentia**

### CODE REVIEW

File: gx-sd-generator.py [hunk 1]
```diff
@@ -168,9 +173,14 @@ def main(argv):
...
         if ofile == "/dev/stdout":
-            print(output(mycloud, myk8s), file=sys.stdout)
+            print(output(mycloud), file=sys.stdout)
         else:
-            print(output(mycloud, myk8s), file=open(ofile, 'a', encoding="UTF-8"))
+            # When output is file type, timestamp and file extension is added to the file name
+            if ostack.outjson:
+                file_name = f'{ofile}_{int(time())}.jsonld'
+            else:
+                file_name = f'{ofile}_{int(time())}.yamlld'
```
Comment (garloff) [inline gx-sd-generator.py:182]: Is "yamlld" something that exists and is well-defined?
I think we output just plain yaml here.
The JSON is different, as we do the headers and @type annotations etc. that make it JSON-LD. (OK, I'm no expert in this.)
Sidenote: We would have expected the user to pass the full filename with extension via the --file/-f option, this is now different, so probably the usage() help output should mention that timestamp and extension are appended to the passed name.

File: gx-sd-generator.py [hunk 2]
```diff
@@ -16,9 +16,14 @@
 import json
 import yaml
 import importlib
+
+from time import time
```
Comment (garloff) [inline gx-sd-generator.py:20]: I think I import time in some subfunction -- if we have a global import, I can remove that piece.
I will push to the branch with that change.

Comment (garloff) [reply gx-sd-generator.py:20]: Done by commit db3cd92.

File: gx-sd-generator.py (hunk 1 above)
Comment (garloff) [reply gx-sd-generator.py:181]: I think we also don't use `.jsonld` elsewhere, but just `.json`. I have no knowledge what is normally done.
(A JSON-LD doc is valid JSON, so naming it `.json` at least does not seem wrong to me. But if `.jsonld` is commonly used elsewhere, e.g. in Gaia-X, I would adapt to that.)

Comment (juanarmentia) [reply gx-sd-generator.py]: Both are accepted, but already changed to json because is more extended and supported by tools/systems

Comment (garloff) [reply gx-sd-generator.py]: Mention that we attach `_time.yaml/json` in help text `usage()`: a5241b3

File: results/osdescriptor_1683188130.jsonld [hunk 3]
```diff
@@ -0,0 +1,1178 @@
+{
```
[the remark this answers is not in the document]
Comment (garloff) [reply results/osdescriptor_1683188130.jsonld]: Either you push a commit message only or we just add the text when we do the squash-merge, as we edit the merge commit message anyway.

**Pull request merged**